src/machine learning analysis.py

```diff
@@ -0,0 +1,170 @@
+# author: Siqi Tao
+# date: 2021-11-26
+
+"""Downloads data as a csv file from online to a local filepath via a URL.
+Usage: download_script.py --url=<url> --filepath=<filepath> 
+ 
+Options:
+--input=<input>       The path or filename pointing to the data
+--prefix=<prefix>     The prefix where to write the output figure(s)/table(s) to and what to call it 
+"""
+
+import os
```

Comment:
Not sure what this is, might not need it, might not even be in environment

Reply:
> 

I think it has something to do with the system and I left it there because I wasn't sure if it would affect saving tables.